test/client/spec/components/types-title.spec.jsx

```diff
@@ -1,48 +0,0 @@
-/**
```

Comment:
This is our project for _teaching_ testing in addition to app development, so we really can't remove the tests....